docs/reference/6-data-import/getting-started/installation/powershell-installation-for-data-import-using-nuget-packages.md

```diff
@@ -0,0 +1,209 @@
+
+![](https://edfidocs.blob.core.windows.net/$web/img/reference/data-import/getting-started/DataImportSqlUser.png)
+
+If you wish to set up your login differently, some basic guidance on how to set this up is explained in this [blog post](https://blogs.msdn.microsoft.com/ericparvin/2015/04/14/how-to-add-the-applicationpoolidentity-to-a-sql-server-login/). You can choose either Windows authentication or SQL Server authentication here. If you choose SQL Server authentication, make sure that if you have "User must change password at next login" checked, you must connect to SSMS with those credentials to reset the password. Otherwise, the app pools won't be able to connect. Also on the Server Roles page, make sure that at least the "dbcreator" checkbox is checked, since Entity Framework will create the database when the application is launched. Once you have confirmed a proper SQL Server login exists, continue to the next step.
+
+## **Step 6. Update Application Pool Identity (Optional)**
+
+As mentioned on Step 5, installation process sets up an appropriate SQL Login for use with the dedicated Data Import Application Pool in IIS. If you would like to use the default "ApplicationPoolIdentity", then you can skip this bit.
+
+Else in the same Advanced Settings window, click on the browse icon under Process Model > Identity. We'll choose the custom account option and click "Set...". When setting the credentials, you can just use the username and password that you use to log in to Windows. If you need to include the app pool domain in the username, then the username can look something like this: "localhost\\username", where "localhost" is the app pool domain. Once we have entered the correct credentials, we'll click OK on all screens until we're back to the main Application Pools page.
+
+![](https://edfidocs.blob.core.windows.net/$web/img/reference/data-import/getting-started/image2019-7-17_16-8-37.png)
+
+# **Launch the Application**
+
+In IIS Manager, click on the DataImport app and select Browse application to launch in the browser. The initial installation will take a minute since the database is being created.
+
+Proceed to instructions in the [Configuration](https://edfi.atlassian.net/wiki/display/EDFITOOLS/First-Time+Configuration) section.
+
+![](https://edfidocs.blob.core.windows.net/$web/img/reference/data-import/getting-started/image2019-7-16_12-58-41.png)
+
+**Once you see the log in screen above, proceed with [Configuration](https://edfi.atlassian.net/wiki/display/EDFITOOLS/First-Time+Configuration).**
+
+# **Installation Errors**
+
+If an error is encountered during installation, use the output from the PowerShell install command for information on the cause of the error. If you require support, please open a ticket on [https://tracker.ed-fi.org/projects/EDFI](https://tracker.ed-fi.org/projects/EDFI) and include this output.
```

Comment:
Another Tracker --> Community Hub example. There are probably others in the repository that I didn't catch; I'll need to do a sweep through looking for others.